src/main/cpp/auton/drivePrimitives/DrivePathPlanner.cpp

```diff
@@ -187,47 +230,41 @@ bool DrivePathPlanner::IsDone()
 void DrivePathPlanner::CheckForDriveToReefBranch()
 {
     // Need to check if there is a Reef Branch
```

Comment:
delete comment isn't really accurate and will become less accurate as we progress, I think